sql/hive/src/main/scala/org/apache/spark/sql/hive/HiveExternalCatalog.scala

```diff
@@ -401,6 +401,13 @@ private[spark] class HiveExternalCatalog(conf: SparkConf, hadoopConf: Configurat
     }
   }
 
+  override def alterTableStats(tableDefinition: CatalogTable): Unit = withClient {
```

Comment:
Now I remember why I add this method. If we've already run the analyze command, then we have Statistics having rowCount, say 100. Now we run alter table command to set "numRows" to 200 (without setting STATS_GENERATED_VIA_STATS_TASK to true). If we use the same alterTable method, since Statistics is defined, we will set numRows and STATS_GENERATED_VIA_STATS_TASK  into properties, just like we do for analyze command.
Then, the parameters in Hive are numRows=100, STATS_GENERATED_VIA_STATS_TASK=true, COLUMN_STATS_ACCURATE=true, while the expected result would be numRows=200, STATS_GENERATED_VIA_STATS_TASK=false, COLUMN_STATS_ACCURATE=false